src/Middleware.php

```diff
@@ -186,7 +186,24 @@ public static function retry(
             return new RetryMiddleware($decider, $delay, $handler, $stats);
         };
     }
-
+    /**
+    * Middleware wrapper function that adds an invocation id header to requests.
+    * This is a uniquely generated UUID to identify initial and subsequent retries as part of a complete request lifecycle.
```

Comment:
This documentation block should point out that the middleware can only be used after the `build` step in a handler stack. Also, please limit all lines to 80 or fewer characters.